content/learning-paths/smartphones-and-mobile/function-multiversioning/_review.md

```diff
@@ -0,0 +1,29 @@
+---
+review:
+    - questions:
+        question: >
+            What is the main benefit of Function Multiversioning?
+        answers:
+            - I may reuse my binaries on different targets without sacrificing runtime performance.
+            - My application binaries will be smaller in size.
+        correct_answer: 1
+        explanation: >
+            The produced binaries can be reused on different targets, but they may be larger in size.
+
+    - questions:
+        question: >
+            Can I implement versions of a function in separate translation units?
+        answers:
+            - Yes, function versions can spread across different translations units.
+            - No, all of them must be in the same translation unit.
```

Comment:
We did that for clang, but (and I don't know if you care...) did gcc get the same feature?

Reply:
We do care about GCC, this learning path is for both toolchains. We made [this design decision](https://github.com/ARM-software/acle/pull/328/files) part of ACLE so GCC will have to adhere to it. Adding @andrewcarlotti for more visibility.